examples/embedding/embedding-test.jl

```diff
@@ -3,6 +3,11 @@
 # tests the output of the embedding example is correct
 using Base.Test
 
+if is_windows()
+    # libjulia needs to be in the same directory as the embedding executable
```

Comment:
Feel free to update the makefile to also copy libjulia if you feel that it's also useful to do that for external users copying the makefile